src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerFileReference.cs

```diff
@@ -417,16 +399,14 @@ public bool Equals(AnalyzerReference other)
 
         public override int GetHashCode()
         {
-            return Hash.Combine(this.Display,
-                        Hash.Combine(this.FullPath, this.IsUnresolved.GetHashCode()));
+            return Hash.Combine(this.Display, this.FullPath.GetHashCode());
         }
 
         public Assembly GetAssembly()
```

Comment:
Ah, I see that it is being used here: http://source.roslyn.io/#Microsoft.CodeAnalysis.Features/CodeFixes/CodeFixService.ProjectCodeFixProvider.cs,52